kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -458,6 +458,13 @@ public class BigQuerySinkConfig extends AbstractConfig {
           + "older than the specified partition expiration time will be permanently deleted. "
           + "Existing tables will not be altered to use this partition expiration time.";
 
+  public static final String GCP_CONNECTOR_PROVIDER_CONFIG = "provider";
```

Comment:
Let's add a comment on top of what this config does since we don't have a corresponding Doc string associated with it.